README.md

```diff
@@ -2,6 +2,9 @@ Discord Soundbot
 ================
 
 [<img title="Join us on Discord" src="https://discordapp.com/api/guilds/236732117524938754/widget.png?style=shield">](https://discord.gg/JBw2BNx)
+[![Docker Pulls](https://img.shields.io/docker/pulls/markokajzer/discord-soundbot.svg?style=shield)](https://hub.docker.com/r/markokajzer/discord-soundbot/tags/)
+[![](https://images.microbadger.com/badges/image/markokajzer/discord-soundbot.svg?style=shield)](https://microbadger.com/images/markokajzer/discord-soundbot)
```

Comment:
Don't really fancy this one. Which uber :neckbeard: would be interested in how many layers there are? 🤓 

What do you think about removing this one?